content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -0,0 +1,416 @@
+| `wdir`           | Working directory for the stage command to run in (relative to the file's location). Any paths in other fields are also based on this. It defaults to `.` (the file's location).                                                                                                          |
+| `deps`           | List of <abbr>dependencies</abbr> paths of this stage (relative to `wdir`).                                                                                                                                                                                                               |
+| `outs`           | List of <abbr>outputs</abbr> paths of this stage (relative to `wdir`). See [Output entries](#output-entries) for more details.                                                                                                                                                            |
+| `params`         | List of <abbr>parameter</abbr> dependency keys (field names) to track from `params.yaml` (in `wdir`). The list may also contain other parameters file names, with a sub-list of the param names to track in them.                                                                         |
+| `metrics`        | List of [metrics files](/doc/command-reference/metrics), and optionally, whether or not this metrics file is <abbr>cached</abbr> (`true` by default). See the `--metrics-no-cache` (`-M`) option of `dvc run`.                                                                            |
+| `plots`          | List of [plot metrics](/doc/command-reference/plots), and optionally, their default configuration (subfields matching the options of `dvc plots modify`), and whether or not this plots file is <abbr>cached</abbr> ( `true` by default). See the `--plots-no-cache` option of `dvc run`. |
+| `frozen`         | Whether or not this stage is frozen from reproduction                                                                                                                                                                                                                                     |
+| `always_changed` | Whether or not this stage is considered as changed by commands such as `dvc status` and `dvc repro`. `false` by default                                                                                                                                                                   |
+| `meta`           | (Optional) arbitrary metadata can be added manually with this field. Any YAML contents is supported. `meta` contents are ignored by DVC, but they can be meaningful for user processes that read or write `.dvc` files directly.                                                          |
```

Comment:
"contents" should be singular ("content").

Reply:
Fixed.